### DISCUSSION TITLE
Build doesn't produce styling

### DISCUSSION THREAD

**Discussion by jasper-zanjani** [squidfunk/mkdocs-material#2540] [Q&A]
My site appears to be hosted properly when using `mkdocs serve`, but using `mkdocs build` and hosting using nginx produces completely unstyled pages. However, developer tools for multiple browsers shows that both JS and CSS files are loaded from the assets subdirectory.

I'm hoping this is a newbie mistake and didn't want to clutter up the Issues page of mkdocs or mkdocs-material. Help!

**Comment by squidfunk** [comment]
Did you set `site_url`?

**Reply by jasper-zanjani** [reply]
Yes I set it to http://localhost, but I'm not sure why that would matter anyway.

**Reply by jasper-zanjani** [reply]
If it helps, please see my [mkdocs.yml](https://github.com/jasper-zanjani/notes/blob/master/mkdocs.yml)
